src/index.js

```diff
@@ -5,12 +5,76 @@ import App from "./containers/App";
 import * as serviceWorker from "./serviceWorker";
 import "./i18n";
 import "bootstrap/dist/css/bootstrap.min.css";
+import { createStore, applyMiddleware, compose } from "redux";
+import rootReducer from "./reducers/rootReducer";
+import { Provider, useSelector } from "react-redux";
+import thunk from "redux-thunk";
+import {
+  reduxFirestore,
+  getFirestore,
+  createFirestoreInstance,
+} from "redux-firestore";
+import {
+  ReactReduxFirebaseProvider,
+  getFirebase,
+  isLoaded,
+} from "react-redux-firebase";
+import firebaseConfig from "./firebaseConfig";
+import firebase from "firebase/app";
+import { Container } from "react-bootstrap";
 
-ReactDOM.render(
-  <Suspense fallback={<div>Loading...</div>}>
-    <App />
-  </Suspense>,
+const store = createStore(
+  rootReducer,
+  compose(
+    applyMiddleware(thunk.withExtraArgument({ getFirebase, getFirestore })),
+    reduxFirestore(firebase, firebaseConfig)
+  )
+);
+
+//construct required properties
+const profileSpecificProps = {
+  userProfile: "profiles",
+  useFirestoreForProfile: true,
+  enableRedirectHandling: false,
+  resetBeforeLogin: false,
+};
+
+//second config propery added
+const rrfProps = {
+  firebase,
 
+  config: firebaseConfig,
+  //second config added here
+  config: profileSpecificProps,
+
+  dispatch: store.dispatch,
+  createFirestoreInstance,
+};
+
+const AuthIsLoaded = ({ children }) => {
+  const auth = useSelector((state) => state.firebase.auth);
+  if (!isLoaded(auth)) {
+    return (
+      <Container>
+        <h4>Loading...</h4>
```

Comment:
can you write a comment about what's going on here? code needs a few more comments. for example, why is there `Loading...` on both line 59 and line 71?